src/components/CustomDropdown/DropdownMenuItems.tsx

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import { Link, LinkProps } from "react-router-dom";
+import styled, { css } from "styled-components";
+import { ArrowRight } from "../Icons";
+
+// Dropdown items can be of a few component
+// varieties, including buttons, router links, and
+// anchor tags. This file unifies their styling,
+// and then divides them into three component exports
+// for semantically explicit usage in dropdowns.
+
+// Common styles
+const baseStyles = css`
+    font-family: ${(props) => props.theme.typography};
+    background: none;
+    border: 2px solid ${({ theme }) => theme.colors.dropdown.background};
+    border-radius: 3px;
+    color: ${({ theme }) => theme.colors.dropdown.text};
+    cursor: pointer;
+    height: 28px;
+    padding: 4px;
+    width: 100%;
+    font-size: 14px;
+
+    &&& {
```

Comment:
This is a nice way in `styled-components` to increase specifiicty, without chasing down lots of antd class names, and without using `!important` which overlooks specificity entirely.